spec.mediawiki

```diff
@@ -487,6 +585,127 @@ reconnect or to act at the appropriate time is considered a violation of
 [[#Rules_of_Community_Conduct|community conduct rules]], so can result in 
 penalties.
 
+===Order Book Subscriptions===
+
+An order book can be viewed and tracked by subscribing to a market. 
+
+'''JSON-RPC method:''' <code>orderbook</code>, '''originator: ''' client
+
+{|
+! field       !! type   !! description
+|-
+| basemarket  || string || currency code for the market's base asset
+|-
+| quotemarket || string || currency code for the market's quote asset
+|}
+
+The response will contain the complete market order book. 
+The order book and all updates include a '''sequence ID''', which increments by 
++1 for every valid order the DEX receives. The client is responsible for 
+tracking the sequence ID to ensure all order updates are received. If an update
+appears to be missing, the client should resubscribe to the market to 
+synchronize the order book from scratch.
+
+'''Response'''
+
+{|
+! field       !! type   !! description
+|-
+| book  || object || An OrderBook object
+|-
+| seq   || int    || A sequence ID
+|}
+
+'''JSON OrderBook object'''
+
+{|
+! field  !! type !! description
+|-
+| bookid || int  || an order book ID, included with future updates
+|-
+| epoch  || int  || the current epoch 
+|-
+| orders || &#91;object&#93; || A list of Order objects
+|}
+
+'''JSON Order object'''
+
+{|
+! field   !! type   !! description
+|-
+| oid     || string || the order ID
+|-
+| otype   || string || "l" for ''limit'', "m" for ''market''
+|-
+| side    || string || "b" for ''buy'', "s" for ''sell''
+|-
+| osize   || int    || order size (atoms)
+|-
+| rate    || int    || price rate (atoms per unit counter-asset). only on limit orders
```

Comment:
should this be quote asset instead of counter-asset?  I'm not sure I'm reading it right...